Content.Server/Glue/GlueSystem.cs

```diff
@@ -35,6 +35,14 @@ private void OnInteract(EntityUid uid, GlueComponent component, AfterInteractEve
             if (!HasComp<ItemComponent>(target))
                 return;
 
+            if (HasComp<GluedComponent>(target))
+            {
+                _popup.PopupEntity(Loc.GetString("glue-failure", ("target", Identity.Entity(target, EntityManager))), args.User,
+                args.User, PopupType.Medium);
+                return;
+            }
+
+
             if (!HasComp<GluedComponent>(target) || !HasComp<UnremoveableComponent>(target))
```

Comment:
This `HasComp<GluedComponent>()` is redundant now.

Reply:
reworked now has an if (HasComp<itemcomponent>()) instead.